projects/cel-go/fuzz_eval.go

```diff
@@ -0,0 +1,40 @@
+package cel
+
+import (
+	"github.com/golang/protobuf/proto"
+
+	"github.com/google/cel-go/checker/decls"
+)
+
+func FuzzEval(data []byte) int {
+	gen := &FuzzVariables{}
+	err := proto.Unmarshal(data, gen)
+	if err != nil {
+		panic("Failed to unmarshal LPM generated variables")
+	}
+
+	env, err := NewEnv()
+	if err != nil {
+		panic("impossible to create env")
+	}
+	for k, _ := range gen.Inputs {
+		env, err = env.Extend(Declarations(decls.NewVar(k, decls.String)))
```

Comment:
In theory you could build this into a list of `[]*exprpb.Decl` values and then call `env, err := cel.NewEnv(cel.Declarations(decls...))` instead of extending the environment.

Reply:
Thanks for the hint, trying this